packages/contracts/test/diamond/facets/UbiquityPoolFacet.t.sol

```diff
@@ -692,6 +692,19 @@ contract UbiquityPoolFacetTest is DiamondTestSetup {
         assertEq(info.redemptionFee, 20000);
     }
 
+    function testAddCollateralToken_ShouldRevertIfCollateralAlreadyExists()
```

Comment:
let's reduce a bit this name, we don't need to add "Already" just `testAddCollateralToken_ShouldRevertIfCollateralExists()`

or

`testAddCollateralToken_RevertOnExistingCollateral`

Reply:
@molecula451 agree, I am on it.

Reply:
less is more

Reply:
done